internal/grpc/grpc.go

```diff
@@ -281,12 +281,14 @@ func (s *Service) Invoke(id, activeEnvironmentID string) (*Response, error) {
 
 	var respHeaders, respTrailers metadata.MD
 
-	timeOut := 5000 * time.Millisecond
+	// Apply timeout if specified (0 means no timeout)
```

Comment:
What if we just increase the initial timeout value to a big value?, like a day!


```suggestion
	timeOut := 12 * time.Hour
```